resources/src/effect/account_thresholds_updated.rs

```diff
@@ -0,0 +1,46 @@
+/// This effect can be the result of a set options operation and represents
+/// the fact that an account's weight thresholds have changed.
+#[derive(Debug, Deserialize)]
+pub struct AccountThresholdsUpdated {
+    account: String,
+    low_threshold: u32,
```

Comment:
Could we possibly rename these to just `low` `medium` and `high` since threadshold is in the struct name?